nattka/bugzilla.py

```diff
@@ -73,6 +74,13 @@ def to_products_components(cls,
             assert False, f'Incorrect BugCategory: {val}'
 
 
+class BugRuntimeTestingState(enum.Enum):
+    UNSET = "---"
```

Comment:
I'd say start by removing this magic member and make it into `typing.Optional[] = None` instead.

Reply:
Done